### PR TITLE
Updating stand-up and github projects

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): Week 1 - Lesson

## Description

 Added the information about Github projects and that they should read each others stand-up

## Who needs to know about this?

 @MarianaVazquezCYF @martaesn @AliciaE08 @ShaunCYFSA

### CODE REVIEW

File: docs/finalproject/week-1/lesson.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ _Facilitated by Tech Assistants_
 
 Every day you should post a message in the team's Slack channel to let the rest of the team know what you’re working on, what progress you’ve made (even, and perhaps _especially_, if it’s none so far) and share anything that’s blocking you.
 
+Don't forget to also read your team mates updates. This way you can understand what is happening in the team and support anyone that needs it.
```
Comment (SallyMcGrath) [inline docs/finalproject/week-1/lesson.md]:
```suggestion
Don't forget to also read your teammates updates. This way you can understand what is happening in the team and support each other.
```

https://www.macmillandictionary.com/dictionary/british/teammate

File: docs/finalproject/week-1/lesson.md [hunk 2]
```diff
@@ -88,6 +88,8 @@ _Facilitated by Tech Assistants_
...
+
 Set a time for this and stick to it.
```
Comment (SallyMcGrath) [inline docs/finalproject/week-1/lesson.md]:
```suggestion
Set a time for this and stick to it. Put it in your calendar.
```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Minor changes

**Comment by kfklein15** [comment]
All accepted, @SallyMcGrath , Just need your final approval. Thank you

**Review by SallyMcGrath — APPROVED** [review]
LGTM 🎉

**Pull request merged**